tests/migrations/2024_04_08_113433_test_new_format.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+use Elastic\Adapter\Indices\Mapping;
+use Elastic\Adapter\Indices\Settings;
+use Elastic\Migrations\Facades\Index;
+use Elastic\Migrations\MigrationInterface;
+
+final class TestNewFormat implements MigrationInterface
```

Comment:
Shouldn't it be `return new class implements MigrationInterface`?

Reply:
Yes, you are right! I'll fix that.